ui/src/timeMachine/components/TimeMachineAlerting.tsx

```diff
@@ -23,6 +25,9 @@ const TimeMachineAlerting: FunctionComponent = () => {
             justifyContent={JustifyContent.FlexEnd}
             margin={ComponentSize.Small}
           >
+            <FeatureFlag name="customCheckQuery">
```

Comment:
more of a personal question: why use the component over the function (`isFlagEnabled`) that the component calls? what abstraction is the component providing over just calling the function directly here?

Reply:
It reads better to me visually, but I do wonder if this makes it evaluate, even though it does not render. testing.